src/Run/CommandLine.cs

```diff
@@ -762,8 +762,7 @@ public string GetHelp(int maxLineWidth, string parameterSetName = null, bool dis
                     }
                 }
             }
-
-            if (hasQualifiers)
+            else if (hasQualifiers)
```

Comment:
Are these mutually exclusive options (parameters vs qualifiers)?  If parameters aren't supported, may want to consider throwing an exception for "hasParameters".  If parameters and qualifiers are both legit options, then maybe change this to 

`if (hasQualifiers)
{
  ...
}
if(!hasQualifiers && !hasParameters)`